<launch>
  <arg name="model" default="$(find yaskawa_cross_modal)/urdf/mygripper.xacro"/>
  <arg name="gui" default="true"/>
  <arg name="rvizconfig" default="$(find yaskawa_cross_modal)/rviz/urdf.rviz"/>
  <param name="robot_description" command="$(find xacro)/xacro $(arg model)"/>
  <param name="use_gui" value="$(arg gui)"/>
  <node name="joint_state" pkg="yaskawa_cross_modal" type="joint_state"/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
  <node name="rviz" pkg="rviz" args="-d $(arg rvizconfig)" type="rviz" required="true"/>
  <!-- <node name="rviz" pkg="rviz" type="rviz" required="true"/> -->
</launch>
